src/core_modules/capture-core/components/Pages/MainPage/WorkingLists/epics/eventsQueryArgsBuilder/filterConverters/dateConverter.js

```diff
@@ -19,7 +21,7 @@ const periods = {
 };
 
 const selectors = {};
-const formatDateForFilterRequest = (dateMoment: moment$Moment) => dateMoment.format('YYYY-MM-DD');
+const formatDateForFilterRequest = (dateMoment: moment$Moment) => getFormattedStringFromMomentUsingEuropeanGlyphs(dateMoment, 'YYYY-MM-DD');
```

Comment:
This follows my previous [comment](https://github.com/dhis2/capture-app/pull/534/files#r409663542).

If we have the default value on the function signature now  we can simply use `getFormattedStringFromMomentUsingEuropeanGlyphs ` instead of `formatDateForFilterRequest ` and this way we have a function less to care about.

Reply:
👍